code/game/objects/items/tcg/tcg.dm

```diff
@@ -167,7 +189,7 @@ GLOBAL_LIST_EMPTY(tcgcard_radial_choices)
 /obj/item/tcgcard_deck
 	name = "Trading Card Pile"
 	desc = "A stack of TCG cards."
-	icon = DEFAULT_TCG_DMI_ICON
+	icon = 'icons/obj/tcgmisc.dmi'
```

Comment:
this was part of the rsc thing that cyber did right? we might wanna keep it as the define.

Reply:
don't remember to be honest

Reply:
I moved non-card icons (packs, decks, binders etc) out of the runtime folder and into a new file for better organisation.